rocketchat-core/build.gradle

```diff
@@ -72,6 +72,7 @@ dependencies {
     compile project (':rocketchat-common')
 //    compile 'com.rocketchat.common:rocketchat-common:0.7.2'
 
+//    compile 'com.rocketchat.common:rocketchat-common:0.7.1.3'
```

Comment:
Duplicate declaration of versioned rocketchat-common

Reply:
will remove it